core/src/banking_stage/consumer.rs

```diff
@@ -64,6 +66,8 @@ pub struct ExecuteAndCommitTransactionsOutput {
     pub commit_transactions_result: Result<Vec<CommitTransactionDetails>, PohRecorderError>,
     pub(crate) execute_and_commit_timings: LeaderExecuteAndCommitTimings,
     pub(crate) error_counters: TransactionErrorMetrics,
+    pub(crate) min_prioritization_fees: usize,
```

Comment:
nit: why `usize` not `u64` as cu_price is defined? Can avoid converting `as usize` below few times